src/stats/bw-readable-stream.js

```diff
@@ -5,7 +5,7 @@ const pump = require('pump')
 const transformChunk = require('./bw-util')
 
 module.exports = (send) => {
-  return (hash, opts) => {
+  return (opts) => {
```

Comment:
Hm.. how doesn't this change break tests?

Reply:
No. Although the build failed due to 'JavaScript heap out of memory'.

Reply:
got it